website/docs/events/m1.md

```diff
@@ -40,7 +42,273 @@ Please consider topics such as:
+Before this, the labs were usually structured with the theoretical part at the beginning and the practical part at the end, so students would just dive into the exercises without reading the theoretical part.
+
+**RD:** The TA should answer questions.
+
+**RD:** If you answer the same questions for 10 students, the answer might need to be in the content.
+
+**RN:** For courses that are technical, exercises should be graded automatically.
+
+**RN:** Ideally, the practical sessions should not be part of the final grade.
+The practical sessions exist to help student learn, not to grade student work.<br />
+**RD:** This is a suggestion / opinion.
+
+**RN:** TutorWeb generates questions based on previous answers.
+If a student answered correctly a difficult question, then the student will receive a more difficult question.<br />
+**BB:** Does it use AI?<br />
+**RD:** We don't know, it's a good idea.
+It can be integrated.
+
+**RN:** Assignments should be automatically graded as long as it's possible.
+We shouldn't spend extra time on this.
+You should ideally score anything between 0 and 100 points, not only 0, 50, or 100 points.
+
+**RN:** The exam should also be public, since everything is open source.
+This is not very common, but all the questions and answers should pe public, the exam is in a controlled context, student wouldn't have access to the repository while taking the exam.
+
+**RN:** Ideally, the exam should be practical, especially for programming-related courses.
+
+**RN:** We would like to offer cryptocurrency for contributors and for elegant solutions to quizzes / assignments.
+SmileyCoin is a cryptocurrency that doesn't have much value, but can be used by anybody. 
+You can use them to buy educational materials, entries to summer schools, books, etc.).
+
+**RN:** In Iceland, students can use them at the university cafeteria.<br />
+**BB:** It's offensive to offer food to those that are more intelligent.<br />
+**RD:** It's complicated, people would try to violate the system.
+
+### Content
+
+**TD:** Our methodology is based on open education resourcs.
+
+**TD:** The  Rs - retain (everybody has a copy of the repository), reuse (educators can use the materials created by other educators), revise (anybody can create pull requests for improvements), remixe (trainers can mix the exising content with their own for another flavor of the material), redistribute (the iformation can be easily shared).
+This is what we want.
+
+**TD:** We use GitHub.
+This is the most popular platform that integrates `git`.
+We keep everything here.
+It's simple to contriute and review changes.
+
+**TD:** Quizzes are stored in Markdown format and are published in multiple formats (ftera transformation) - Moodle, TutorWeb, etc.
+
+**TD:** A quiz has at least 3 sections- the question itself, the possible answers the feedback (given in case the stuent doesn't answer a questions correctly).
```

Comment:
```suggestion
**TD:** A quiz has at least 3 sections: the question itself, the possible answers the feedback (given in case the stuent doesn't answer a questions correctly).
```